bin/stringfuzzx

```diff
@@ -195,8 +195,10 @@ def main():
         print(e, file=sys.stderr)
         return 1
 
-    # filter out suppressed expressions
-    ast = list(filter(is_not_suppressed, ast))
+    # the nop transformer should not modify anything
+    if not transformer == nop:
```

Comment:
I'm actually unsure whether `not X == Y` is `not (X == Y)` or `(not X) == Y`. It would be clearer to use `transformer != nop`.